.github/workflows/dapr-test.yml

```diff
@@ -535,9 +535,6 @@ jobs:
           make setup-test-env-kafka
           make setup-test-env-zipkin
           make setup-test-env-postgres
-          if [ "${{ env.DAPR_TEST_STATE_STORE }}" == "" ] ; then
-            make setup-test-env-mongodb
```

Comment:
Would we clean all mongo setup:
```
root:[dapr]$ git grep setup-test-env-mongodb
.github/workflows/dapr-test.yml:539:            make setup-test-env-mongodb
.github/workflows/kind-e2e.yaml:197:        make setup-test-env-mongodb
.github/workflows/version-skew-e2e.yaml:222:        make setup-test-env-mongodb
tests/dapr_tests.mk:257:setup-3rd-party: setup-helm-init setup-test-env-redis setup-test-env-kafka setup-test-env-mongodb setup-test-env-zipkin setup-test-env-postgres
tests/dapr_tests.mk:501:setup-test-env-mongodb:
tests/dapr_tests.mk:535:setup-test-env: setup-test-env-kafka setup-test-env-redis setup-test-env-mongodb setup-test-env-postgres setup-test-env-k6 setup-test-env-zipkin setup-test-env-postgres
tests/docs/running-e2e-test.md:116:make setup-test-env-mongodb
```

Reply:
We do not set this to mongodb anymore so this is not needed.